keystrokes/src/components/Character/CharacterDetails/CharacterDetailPage.js

```diff
@@ -72,8 +72,19 @@ function CharacterDetailPage({ userId, match }) {
 		}
 
 		const error = await ref.update(updates);
+		setAlertMessage(error);
+	}
+
+	async function selectCharacter() {
+		let updates = {};
+		updates[`/Users/${userId}/selectedCharacter`] = characterId;
+		const error = await ref.update(updates);
```

Comment:
since there's only one update do you need to send it an object?

Reply:
i wasnt actually sure how to do it without the object. i figured i threw in the path into update, but i wasnt sure where to put characterId